indico/modules/events/registration/forms.py

```diff
@@ -454,6 +454,11 @@ def validate_user_principals(self, field):
 
 
 class BadgeSettingsForm(IndicoForm):
+    fieldsets = (
```

Comment:
We use `_fieldsets` in other places for almost the same thing so I'd do the same here.